primitives/npos-elections/src/mock.rs

```diff
@@ -226,10 +226,10 @@ where
 	if backing_backed_stake.len() > 0 {
 		let max_stake = backing_backed_stake
 			.iter()
-			.max_by(|x, y| x.partial_cmp(&y).unwrap_or(sp_std::cmp::Ordering::Equal))
+			.max_by(|x, y| x.partial_cmp(y).unwrap_or(sp_std::cmp::Ordering::Equal))
```

Comment:
This seems like an unrelated change to me too.

Reply:
similar error as above

Reply:
Aha, ok. As this is only a mock, I don't think they care that we have introduced a copy here.